src/components/app/data/services/enterpriseCustomerUser.js

```diff
@@ -204,3 +204,17 @@ export async function updateUserCsodParams({ data }) {
   const url = `${getConfig().LMS_BASE_URL}/integrated_channels/api/v1/cornerstone/save-learner-information`;
   return getAuthenticatedHttpClient().post(url, data);
 }
+
+/**
+ * Helper function to unlink an enterprise customer user by making a POST API request.
+ * @param {string} enterpriseCustomerUserUUID - The UUID of the enterprise customer user to be unlinked.
+ * @param {string} userEmail - The email of the user that should be unlinked.
+ * @returns {Promise} - A promise that resolves when the user is successfully unlinked from the enterprise customer.
+ * @throws {Error} - Throws an error if the unlinking process fails.
```

Comment:
nit: given the use of `catch` (via the current `.catch` or the recommended `try`/`catch`), I don't believe this function actually throws errors for callers of it?

Reply:
@jajjibhai008 Yes, I understand the intent for the `catch`. I'm just clarifying that the function doesn't actually throw an error to its _callers_ because the function catches the error. Given this, the JSDoc comment denoting that the function throws an error is misleading.